iaso/api/org_unit_change_requests/views.py

```diff
@@ -218,20 +247,150 @@ def export_to_csv(self, request):
         writer.writerow(self.CSV_HEADER_COLUMNS)
 
         for change_request in filtered_org_unit_changes_requests:
+            # Helper function to determine if a field has changed
+            def get_conclusion(old_value, new_value):
+                if old_value == new_value:
+                    return "same"
+                return "updated"
+
+            # Get parent reference extensions
+            def get_parent_ref_ext(parent, level):
+                if not parent:
+                    return None
+
+                # Get ancestors up to the specified level
+                ancestors = list(parent.ancestors().order_by("path"))
+                if level <= len(ancestors):
+                    return ancestors[level - 1].source_ref
+                return None
+
+            # Get location string
+            def get_location_str(location):
+                if not location:
+                    return None
+                return f"{location.y}, {location.x}"
+
+            # Get reference instance IDs
+            def get_reference_instance_ids(instances):
+                if not instances.exists():
+                    return ""
+                return ",".join(str(instance.id) for instance in instances.all())
+
+            # Basic row data
             row = [
                 change_request.id,
                 change_request.org_unit_id,
                 change_request.org_unit.source_ref,
                 change_request.org_unit.name,
                 change_request.org_unit.parent.name if change_request.org_unit.parent else None,
-                change_request.org_unit.org_unit_type.name,
+                change_request.org_unit.org_unit_type.name if change_request.org_unit.org_unit_type else None,
                 ",".join(group.name for group in change_request.org_unit.groups.all()),
                 change_request.get_status_display(),
```

Comment:
I may be wrong but I think here is the validation_status of the orgUnit and not the change request status